src/test/java/calculator/OperandTest.java

```diff
@@ -0,0 +1,39 @@
+package calculator;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatExceptionOfType;
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class OperandTest {
+
+
+    @Test
+    void negativeValueCheckTest() {
+        String[] str = {"1", "-1", "923"};
+        assertThatExceptionOfType(RuntimeException.class).isThrownBy(
+            () -> Operand.parseOperand(str));
+    }
+
+    @Test
+    void normalParsingOperationTest() {
+        String str = "1,2,3";
+
+        List<Operand> operandList = new ArrayList<>();
+        operandList.add(new Operand(1));
+        operandList.add(new Operand(2));
+        operandList.add(new Operand(3));
+
+        Operands operands = new Operands(Operand.parseOperand(str.split(",")));
+
+        assertThat(operands.operandList.get(0).getOperand()).isEqualTo(
+            operandList.get(0).getOperand());
+        assertThat(operands.operandList.get(1).getOperand()).isEqualTo(
+            operandList.get(1).getOperand());
+        assertThat(operands.operandList.get(2).getOperand()).isEqualTo(
+            operandList.get(2).getOperand());
```

Comment:
`Operand` 에 `hashCode`, `equals` 만 정의해주신다면 다음과 같이 간단하게 검증이 가능할 것 같아요 
참고만해주세요! 😆
```suggestion
                assertThat(operands.operandList).isEqualTo(operandList);
```